whereclause.go

```diff
@@ -92,6 +92,12 @@ func (wc *WhereClause) AddWhereExpr(args *Args, andExpr ...string) *WhereClause
 		return wc
 	}
 
+	andExprsBytesLen := estimateStringsBytes(andExpr)
+
+	if andExprsBytesLen == 0 {
```

Comment:
Need some cases to cover this branch.

Reply:
I've added test covering those lines. But found another problem. 

Now building with `WHERE` clause can produce an extra blank space. In case `WHERE` was added through `AddWhereExpr` function. Since it can be only applied to an existing and non nil `WhereClause` it will be replaced with empty space during Build stage.

This problem can be  reproduced for both empty array and array of empty strings. I've added two test cases `TestEmptyStringsWhereAddWhereExpr` and `TestEmptyAddWhereExpr` to demonstrate how it can be achieved.

I felt that fixing this is not a part of current pr. I think its a bit another problem rather the one i was trying to fix here. I can still try to fix it in a current or maybe in a new pr. What do you think about it?